server/src/controllers/Events.js

```diff
@@ -46,11 +46,23 @@ module.exports = {
 		}
 	},
 	async updateEvent(req, res) {
-		const eventId = req.params.id;
-		const event = await Event.findById(eventId);
-		res.json({
-			"id": event._id
-		});
+		try {
+			const file = req.file;
+			const image = cloudinary.v2.uploader.upload(file.path);
+			req.body.image = {
+				url: image.secure_url,
+				public_id: image.public_id
+			};
+			const eventId = req.params.id;
+			const event = await Event.findByIdAndUpdate(eventId, req.body);
+			res.json({
+				"id": event._id
+			});
+		} catch (err) {
+			res.status(400).send({
```

Comment:
I think the status should be 
1.  404 - Request not found.
2.  500 - Internal error.

Reply:
For status code, I referred to the one used in Blog.js controller file but I'll change it to either 404 or 500.